src/llmcompressor/transformers/tracing/mllama.py

```diff
@@ -0,0 +1,162 @@
+# flake8: noqa
+# coding=utf-8
+# Copyright 2024 the HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# vllm-project: no copyright
+"""PyTorch Mllama model."""
+
+from typing import List, Optional, Tuple, Union
+
+import torch
+import torch.utils.checkpoint
+
+from transformers.modeling_outputs import CausalLMOutputWithPast
+from transformers.utils import (
+    add_start_docstrings,
+    logging,
+)
+
+# TRACING: imports
+from torch.fx import wrap
+from transformers.models.mllama.modeling_mllama import (
+    MLLAMA_START_DOCSTRING,
+    MllamaForConditionalGeneration,
+)
+
+logger = logging.get_logger(__name__)
+
+
+# TRACING: This function is not traceable
```

Comment:
I think it would be helpful to have quick notes to help understand what you had to update to make this model traceable

Reply:
> what you had to update to make this model traceable

"what" I had to update to make this model traceable is exactly the code in this file. "why" and "where" it was done is indicated by the `# TRACING` comments